plugins/backend/catalog-backend-module-okta/src/providers/groupEntityFromOktaGroup.ts

```diff
@@ -20,22 +20,17 @@ import { GroupNamingStrategy } from './groupNamingStrategies';
 export const groupEntityFromOktaGroup = (
   group: Group,
   namingStrategy: GroupNamingStrategy,
+  parentGroup: Group | undefined,
   options: {
     annotations: Record<string, string>;
     members: string[];
-    parentGroupField?: string;
   },
 ): GroupEntity => {
-  const parentFieldValue = options.parentGroupField
-    ? group.profile[options.parentGroupField]
-    : undefined;
-  let parent: string | undefined = undefined;
-  if (typeof parentFieldValue === 'string') {
-    parent = parentFieldValue;
-  }
-  if (typeof parentFieldValue === 'number') {
-    parent = parentFieldValue.toString();
+  let parentGroupName = undefined;
+  if (parentGroup) {
```

Comment:
If you move this down after defining the group entity you can simplify the code and only have one if statement and no variable reassignment

Reply:
done